app/assets/javascripts/metrics_page.es6

```diff
@@ -0,0 +1,108 @@
+(($) => {
+    const $document = $(document);
+
+
+
+    $document.on('turbolinks:load');
+})(window.jQuery);
+
+function hideSecondaryNavItems(){
+    debugger
+    document.getElementById('peSideNavAbout').style.display = 'none';
```

Comment:
can we also add css classes in this file rather than manipulating styles directly on the elements?

Reply:
....classList.add(‘display-none’)

Reply:
got it done with Josh's help!